2-explicit-polymorphism/2-explicit-polymorphism.md

```diff
@@ -0,0 +1,147 @@
+# Introduction
+
+Currently in Hazel every function has to be assigned a monomorphic type.
+This makes it impossible to write code that operates on more than one type
+of data. We currently have an example that implements `map : (Num -> Num)
+-> [Num] -> [Num]`. By adopting this proposal, Hazel's map function can
+have the type `forall a. (a -> a) -> [a] -> [a]`, and thereby be useful for
```

Comment:
map has type `forall a. forall b. (a -> b) -> [a] -> [b]`

Reply:
Fixed in new branch